### ISSUE TITLE
Add support for remapping.

### ISSUE THREAD

**Issue by MicahZoltu** [Latrasis/linter-solidity#15]
My import statements leverage Solidity remapping which means they look like:
```
import 'ROOT/relative/path/to/other.sol
```

I then provide a remapping in my solc input JSON that maps `ROOT` to an absolute path of the root of my solidity source code.

Unfortunately, I don't see a way to provide such a remapping in this package, which means all of my imports are flagged as bad.

**Comment by willclarktech** [comment]
This is also a problem when using Truffle.
